src/components/edit/WithHoverMenu.tsx

```diff
@@ -1,29 +1,24 @@
-import {
-    Button as UnstyledButton,
-    ButtonGroup,
-    Theme,
-    Tooltip as UnstyledTooltip,
-    withStyles,
-} from "@material-ui/core";
+import { Button, ButtonGroup, styled } from "@mui/material";
+import { grey } from "@mui/material/colors";
 import React from "react";
 import { DataTestID } from "../../common/DataTestID";
+import { makeStyledTooltipMenu } from "./StyledTooltip";
 
-const Button = withStyles((theme: Theme) => ({
-    contained: {
-        backgroundColor: "transparent",
-        "&:hover": {
-            backgroundColor: theme.palette.primary.dark,
-        },
+const HoverMenuButton = styled(Button)({
+    backgroundColor: "transparent",
+    "&:hover": {
+        backgroundColor: grey[200],
     },
-}))(UnstyledButton);
+});
 
-const Tooltip = withStyles({
-    tooltip: {
-        padding: 0,
-        background: "transparent",
-        margin: 0,
-    },
-})(UnstyledTooltip);
+const TransparentTooltip = makeStyledTooltipMenu(() => ({
+    padding: "0px",
+    // !important is awful. however MUI keeps overriding the specified style
+    // and there isn't a good guide on how to compose this properly
+    // so that the customized style is observed. be great to fix this in the future
+    margin: "0px !important",
```

Comment:
what the comments say - had some trouble making this style respected and not overridden. this is probably a stable hack since there's not going to be too much more style changes in this area....famous last wods